compiler/src/lib.rs

```diff
@@ -337,7 +339,7 @@ pub fn inputs_to_query_callback<T: FieldElement>(inputs: Vec<T>) -> impl Fn(&str
             }
             "\"print_char\"" => {
                 assert_eq!(items.len(), 2);
-                print!("{}", items[1].parse::<u8>().unwrap() as char);
+                log::info!("{}", items[1].parse::<u8>().unwrap() as char);
```

Comment:
I'm not really sure about this one. This is not the main `ebreak/ecall` thin for riscv - do you know where it is used?

Reply:
Ah no it actually is the main one!

At some point, we should probably store this into a proper buffer so that it does not interfere with the logging.

Reply:
Searching for `print_char`, I only found [this line](https://github.com/powdr-labs/powdr/blob/main/riscv/src/compiler.rs#L1170), for the RISC-V ebreak instruction!

Reply:
OK, I changed it back to print!